src/app.tsx

```diff
@@ -7,6 +7,7 @@
 
 import i18n from 'i18next';
 import caLocale from './locales/ca.json';
+import daLocale from './locales/da.json'
```

Comment:
```suggestion
import daLocale from './locales/da.json';
```
Missing semicolon. I'll merge first thing tomorrow though. Thanks :)